<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/top/images/mainicon.png</normaloff>:/images/top/images/mainicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="3">
     <widget class="QPushButton" name="addFileButtonTrigger">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/top/images/addfile.png</normaloff>:/images/top/images/addfile.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>71</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="6" column="7" rowspan="3" colspan="3">
     <widget class="QGroupBox" name="deleteKeyGroupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>DeleteKey</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Key</string>
       </property>
      </widget>
      <widget class="QComboBox" name="deleteKeyBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteKeyButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>128</y>
         <width>93</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" rowspan="6" colspan="2">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="openDirectoryButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/top/images/folder.png</normaloff>:/images/top/images/folder.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>71</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="5" colspan="3">
     <widget class="QComboBox" name="comboSectionsBox"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="chooseFile">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/top/images/inifile.png</normaloff>:/images/top/images/inifile.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>71</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="9">
     <widget class="QPushButton" name="cancelSectionAddingButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/top/images/cross.png</normaloff>:/images/top/images/cross.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>71</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="saveCurFile">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/top/images/savefile.png</normaloff>:/images/top/images/savefile.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>71</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Section</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4" rowspan="3" colspan="3">
     <widget class="QGroupBox" name="createKeyGroupBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Creating</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Key</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
      <widget class="QComboBox" name="typeEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>int</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bool</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>double</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>string</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="keyEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="valueEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="addButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>93</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="5" column="8">
     <widget class="QPushButton" name="addSectionButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/images/top/images/plus.png</normaloff>:/images/top/images/plus.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>71</width>
        <height>50</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="4" colspan="6">
     <widget class="QTreeWidget" name="documentTreeView">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="8" rowspan="2" colspan="2">
     <widget class="QGroupBox" name="fileAdderBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Add File</string>
      </property>
      <widget class="QLineEdit" name="fileNameInput">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addFileButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="openedFilesInfo">
      <property name="text">
       <string>Files opened: 0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" rowspan="2" colspan="2">
     <widget class="QGroupBox" name="sectionAdderBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Add section</string>
      </property>
      <widget class="QLineEdit" name="sectionNameInput">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addSectionActionButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>93</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
